roles/ceph-monitor/tasks/main.yml

```diff
@@ -77,3 +77,6 @@
             owner=root
             group=root
             mode=0755
+
+- include: monitoring.yml tags=monitoring,common
```

Comment:
I would prefer the style where tags are not a key= argument, but instead part of the task yaml hash.

```
- include: monitoring.yml
  tags:
    - monitoring
    - common
```